src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoSubmissionServiceImpl.java

```diff
@@ -3,15 +3,13 @@
 import ca.bc.gov.ag.csows.filing.NestedEjbException_Exception;
 import ca.bc.gov.ag.csows.filing.ProcessItemStatus;
 import ca.bc.gov.ag.csows.filing.*;
-import ca.bc.gov.ag.csows.filing.status.GetDocumentStatusTypes;
-import ca.bc.gov.ag.csows.filing.status.GetDocumentStatusTypesResponse;
 import ca.bc.gov.ag.csows.services.*;
```

Comment:
If you're cleaning up the imports, one thing you may want to consider is to remove wildcard imports.  Not at all critical, but considered a code smell by some.

https://rules.sonarsource.com/java/tag/pitfall/RSPEC-2208
https://checkstyle.sourceforge.io/config_imports.html#AvoidStarImport

Reply:
It is intellij "Optimizing the imports". I actually didn't relieve that feature got turned back on..... Thanks for catching it. I will have to turn it off again :(